cropper/src/main/res/menu/crop_image_menu.xml

```diff
@@ -1,18 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
-      xmlns:app="http://schemas.android.com/apk/res-auto">
-
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/ic_crop_shape"
+        android:icon="@drawable/ic_crop_shape_square_24"
+        android:title="@string/ic_crop_shape_square"
```

Comment:
We are not updating the title when we update the icon, what can cause confusion when this is showed as text. Please check this =D